### PR TITLE
Home and registry page

### PR DESCRIPTION
**Pull request by kevinsslin**
Adding a home page and a user page, as shown below
image
image

### CODE REVIEW

**Comment by hcc429** [comment]
讚讚讚 還有附圖片

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,48 @@
...
+        <Content style={contentStyle}>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/user" element={<User />} />
+            <Route path="*" element={<h1>Not Found</h1>} />
+          </Routes>
+        </Content>
+        <Footer style={{ textAlign: "center" }}>
+          I'm Uber ©2023 Created by 112-1 Group30
+        </Footer>
```
Comment (hcc429) [inline client/src/App.tsx:43]: Footer 可以移去 Component

Comment (kevinsslin) [reply client/src/App.tsx:43]: Ok 應該等確認一點會再 refactoring XD

**Comment by kevinsslin** [comment]
Adding login and register page
### Login Page
image
### Register Page (while empty input)
image

**Pull request merged**